{"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkError"}]}],"kind":"declarations"}],"identifier":{"url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/networkError","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/rxfireauth\/usererror\/networkerror"]}],"schemaVersion":{"minor":3,"patch":0,"major":0},"sections":[],"metadata":{"symbolKind":"case","roleHeading":"Case","title":"UserError.networkError","externalID":"s:10RxFireAuth9UserErrorO07networkE0yA2CmF","role":"symbol","modules":[{"name":"RxFireAuth"}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkError"}]},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError"]]},"abstract":[{"text":"An error occurred while reaching Firebase servers.","type":"text"}],"references":{"doc://RxFireAuth/documentation/RxFireAuth/UserError/networkError":{"type":"topic","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError\/networkError","title":"UserError.networkError","abstract":[{"text":"An error occurred while reaching Firebase servers.","type":"text"}],"role":"symbol","url":"\/documentation\/rxfireauth\/usererror\/networkerror","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"networkError","kind":"identifier"}],"kind":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth/UserError":{"type":"topic","url":"\/documentation\/rxfireauth\/usererror","role":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/UserError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"UserError"}],"abstract":[{"type":"text","text":"Errors thrown by  "},{"type":"codeVoice","code":"UserManagerType"},{"type":"text","text":" implementations."}],"title":"UserError","navigatorTitle":[{"kind":"identifier","text":"UserError"}],"kind":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","type":"topic","url":"\/documentation\/rxfireauth","kind":"symbol","title":"RxFireAuth","role":"collection","abstract":[]}}}